package-lock.json

```diff
@@ -6543,7 +6543,7 @@
       "dev": true
     },
     "geotiff": {
-      "version": "github:ilan-gold/geotiff.js#498390fd2a95de4808bc1add3266f1e67baeec9b",
+      "version": "github:ilan-gold/geotiff.js#a6f93c1be629d3630f6d506bb98c10ddeaaafc10",
```

Comment:
Was there a change to your fork of geotiff?

Reply:
No, I'll remove this.